src/ambianic/pipeline/ai/image_boundingBox_detection.py

```diff
@@ -7,12 +7,12 @@
 from PIL import ImageOps
 from .inference import TFInferenceEngine
 from ambianic.pipeline import PipeElement
-
+from ambianic.pipeline.ai.tf_detect import TFDetectionModel
 
 log = logging.getLogger(__name__)
 
 
-class TFImageDetection(PipeElement):
+class TFBoundingBoxDetection(PipeElement):
```

Comment:
@bhavikapanara Should TFBoundingBoxDetection extend TFDetectionModel? 
It seems like __ini__, load_labels, and log_stats are reusable.

Reply:
@bhavikapanara did you see this comment ^^^

Reply:
@ivelin Can you please check here: https://github.com/ambianic/ambianic-edge/blob/9915364eaa507f89ad17124125c7b1fb8ffbc211/src/ambianic/pipeline/ai/image_boundingBox_detection.py#L39

Reply:
Right. I see that you chose to use class delegation instead of inheritance. Based on the intended use of these classes and the redundant code in each, I am not sure that delegation is the optimal choice. Maybe you can elaborate to help me understand why inheritance is not desired.

Reply:
@ivelin Actually I faced some error for multiple inheritances at that time. Let me show you

Reply:
```
======================================================== ERRORS =========================================================
________________________________ ERROR collecting tests/pipeline/ai/test_fall_detect.py _________________________________
tests/pipeline/ai/test_fall_detect.py:3: in <module>
    from ambianic.pipeline.ai.object_detect import ObjectDetector
src/ambianic/pipeline/ai/object_detect.py:3: in <module>
    from .image_boundingBox_detection import TFBoundingBoxDetection
src/ambianic/pipeline/ai/image_boundingBox_detection.py:15: in <module>
    class TFBoundingBoxDetection(PipeElement, TFDetectionModel):
E   TypeError: Cannot create a consistent method resolution
E   order (MRO) for bases PipeElement, TFDetectionModel
```

Reply:
@ivelin for that reason I used class delegation instead of inheritance. Please help me to resolve this error.

Reply:
Why do you need multiple inheritance? TFDetectionModel is a PipeElement itself. You can replace 
```
class TFBoundingBoxDetection(PipeElement, TFDetectionModel):
```

with 

```
class TFBoundingBoxDetection(TFDetectionModel):
```

Reply:
@ivelin Ohh...I missed this point...My apology